tests/suit/framework/framework.go

```diff
@@ -137,7 +137,9 @@ func (f *Framework) BeforeEach() {
 		})
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		f.Namespace = ns
-		f.AddNamespaceToDelete(ns)
+
+		//dont delete namespace after tests are done
```

Comment:
or to have a flag that is not set by default for deleting the generated namespace

Reply:
done, added flags `-delete-namespace` which can be added to the ginkgo execution parameter.